clients/src/main/java/org/apache/kafka/clients/consumer/internals/Coordinator.java

```diff
@@ -217,21 +217,6 @@ public void ensureCoordinatorKnown() {
         }
     }
 
-    /**
```

Comment:
@ewencp Is this comment true? I did not find blocking code in `commitOffsetAsync`. It looks it just put the OffsetCommitRequest to the NetworkClient and then return the future. Did I miss anything?